deployment/bicep/backend.bicep

```diff
@@ -0,0 +1,433 @@
+  //sku: {
+    //name: 'Standard_LRS'
+    //tier: 'Standard'
+  //}
+  properties: {
+    changeFeed: {
+      enabled: false
+    }
+    restorePolicy: {
+      enabled: false
+    }
+    containerDeleteRetentionPolicy: {
+      enabled: false
+    }
+    cors: {
+      corsRules: []
+    }
+    deleteRetentionPolicy: {
+      enabled: false
+    }
+    isVersioningEnabled: false
+  }
+}
+
+// Azure File Service for Azure Storage Account
+resource Microsoft_Storage_storageAccounts_fileServices_storageAccounts_saapimcsbackend_name_default 'Microsoft.Storage/storageAccounts/fileServices@2021-04-01' = {
+  parent: storageAccounts_saapimcsbackend_name_resource
+  name: 'default'
+    // mb: this part generates an error on deployment ("sku is read-only")
+  //sku: {
+  //    name: 'Standard_LRS'
+  //    tier: 'Standard'
+  //  }
+  properties: {
+  // mb: this part generates an error on deployment ("XML not valid")
+  //    protocolSettings: {
+  //      smb: {}
+  //    }
+    cors: {
+      corsRules: []
+    }
+    shareDeleteRetentionPolicy: {
+      enabled: false
+      days: 0
+    }
+  }
+}
+
+// Azure Queue Service for Azure Storage Account
+resource Microsoft_Storage_storageAccounts_queueServices_storageAccounts_saapimcsbackend_name_default 'Microsoft.Storage/storageAccounts/queueServices@2021-04-01' = {
+  parent: storageAccounts_saapimcsbackend_name_resource
+  name: 'default'
+  properties: {
+    cors: {
+      corsRules: []
+    }
+  }
+}
+
+// Azure Table Service for Azure Storage Account
+resource Microsoft_Storage_storageAccounts_tableServices_storageAccounts_saapimcsbackend_name_default 'Microsoft.Storage/storageAccounts/tableServices@2021-04-01' = {
+  parent: storageAccounts_saapimcsbackend_name_resource
+  name: 'default'
+  properties: {
+    cors: {
+      corsRules: []
+    }
+  }
+}
+
+// Azure Function App (Linux, .NET Core 3.1)
+resource sites_funcappAPIMCSBackendMicroServiceA_name_resource 'Microsoft.Web/sites@2018-11-01' = {
+  name: sites_funcappAPIMCSBackendMicroServiceA_name
+  location: 'West Europe'
+  kind: 'functionapp,linux'
+  properties: {
+    enabled: true
+    hostNameSslStates: [
+      {
+        name: 'funcappapimcsbackendmicroservicea.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Standard'
+      }
+      {
+        name: 'funcappapimcsbackendmicroservicea.scm.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Repository'
+      }
+    ]
+    serverFarmId: serverfarms_appsvcplanAPIMCSBackend_name_resource.id
+    reserved: true
+    isXenon: false
+    hyperV: false
+    siteConfig: {
+      numberOfWorkers: 1
+      linuxFxVersion: 'dotnet|3.1'
+      alwaysOn: true
+      http20Enabled: false
+    }
+    scmSiteAlsoStopped: false
+    clientAffinityEnabled: false
+    clientCertEnabled: false
+    hostNamesDisabled: false
+    containerSize: 1536
+    dailyMemoryTimeQuota: 0
+    httpsOnly: false
```

Comment:
I'd strongly suggest we invest some time in getting SSL implemented here. For first iteration I'm fine but we cannot have none secure transport over http imho

Reply:
agree: changed to true